odoo/tools/convert.py

```diff
@@ -169,6 +170,8 @@ def _process(s):
                 return [_eval_xml(self, n, env) for n in node.iterchildren('value')]
             case 'tuple':
                 return tuple(_eval_xml(self, n, env) for n in node.iterchildren('value'))
+            case 'bytes':
+                return bytes(data)
```

Comment:
Useless, all that does is raise `TypeError`.

Reply:
Updated to encode strings. I keep forgetting that `str.encode` has a default, but `bytes` does not.